assets/prototype/domain/eventEditor/datesList/dateCard/EditDateModal.js

```diff
@@ -0,0 +1,22 @@
+import { useContext } from '@wordpress/element';
+import FormModal from '../../../shared/FormModal';
+import EditDateForm from '../EditDateForm';
+import { DateTimeContext } from '../../../../infrastructure/services/contextProviders/DateTimeProvider';
+
+const EditDateModal = ({ eventId, tickets, handleClose }) => {
+	const { isOpen, setIsOpen } = useContext(DateTimeContext);
+	const onClose = () => setIsOpen(false);
```

Comment:
why not move this into `DateTimeProvider` as well ?

Reply:
sure, thx